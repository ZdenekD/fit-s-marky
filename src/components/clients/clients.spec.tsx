import {shallow, ShallowWrapper} from 'enzyme';
import findComponent from '../../__test__/utils/helpers';
import Clients from '.';

describe('Clients', () => {
    let wrapper: ShallowWrapper;

    beforeEach(() => {
        wrapper = shallow(<Clients />);
    });

    it('match snapshot', () => {
        expect(wrapper.html()).toMatchSnapshot();
    });

    it('renders without error', () => {
        const component = findComponent(wrapper, 'component-clients');

        expect(component.exists()).toBe(true);
    });
});
