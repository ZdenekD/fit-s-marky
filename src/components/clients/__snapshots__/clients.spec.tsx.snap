// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Clients match snapshot 1`] = `
<section class="section"
         data-test="component-clients"
         style="opacity:0"
>
  <h3 class="heading">
    Spokojení klienti:
  </h3>
  <ul class="list">
    <li class="item">
      <img src="/clients/johnson&amp;johnson.png"
           alt="Johnson&amp;Johnson"
           class="image"
      >
      <br>
      <span class="title">
        Johnson&amp;Johnson
      </span>
    </li>
    <li class="item">
      <img src="/clients/caerus_im.png"
           alt="Caerus IM"
           class="image"
      >
      <br>
      <span class="title">
        Caerus IM
      </span>
    </li>
    <li class="item">
      <img src="/clients/vodafone.png"
           alt="Vodafone"
           class="image"
      >
      <br>
      <span class="title">
        Vodafone
      </span>
    </li>
    <li class="item">
      <img src="/clients/paralelni_polis.png"
           alt="Paralelní Polis"
           class="image"
      >
      <br>
      <span class="title">
        Paralelní Polis
      </span>
    </li>
    <li class="item">
      <img src="/clients/form_factory.png"
           alt="Form Factory"
           class="image"
      >
      <br>
      <span class="title">
        Form Factory
      </span>
    </li>
    <li class="item">
      <img src="/clients/pilates&amp;flow.png"
           alt="Pilates &amp; Flow"
           class="image"
      >
      <br>
      <span class="title">
        Pilates &amp; Flow
      </span>
    </li>
    <li class="item">
      <img src="/clients/joga_letna.png"
           alt="Joga Letná"
           class="image"
      >
      <br>
      <span class="title">
        Joga Letná
      </span>
    </li>
    <li class="item">
      <img src="/clients/fitness_flora.png"
           alt="HitFit Flora"
           class="image"
      >
      <br>
      <span class="title">
        HitFit Flora
      </span>
    </li>
    <li class="item">
      <img src="/clients/k6.png"
           alt="Pohybové studio K6"
           class="image"
      >
      <br>
      <span class="title">
        Pohybové studio K6
      </span>
    </li>
    <li class="item">
      <strong class="title">
        TJ Sokol Jahodnice
      </strong>
    </li>
  </ul>
</section>
`;
